app/src/public/css/scoreboard.css

```diff
@@ -0,0 +1,11 @@
+.container {
+    display: grid;
+    place-items: center;
+    width: 100%;
+}
+
+
+
+.col-sm-2 {
+    text-align: center;
+}
```

Comment:
Don't do this - you are overriding a bootstrap class. Create your class (or use bootstrap's `text-center` class, that is, where you have `class="col-sm-2"` change it to `class="col-sm-2 text-center"`)